code/game/objects/items/explosives/mine.dm

```diff
@@ -293,6 +384,76 @@
 	iff_signal = null
 
 
+// Subtype from this to create buried mines.
+/obj/item/explosive/mine/bury
+	use_tripwire = FALSE
+	needs_digging = TRUE
+	map_prespawn = FALSE
+	buried = FALSE
+	arming_time = 3 SECONDS
+	use_dir = FALSE
+	unacidable = TRUE
+	var/datum/effect_system/spark_spread/sparks = new
```

Comment:
no static new-s, put that in Initialize or better yet only new it when it's used, given it's deleted in disarm() and prime()

Reply:
What?

Reply:
you can 't do `= new` here statically, new it where appropriate to use